validation-test/compiler_crashers_fixed/26813-generic-enum-tuple-optional-payload.swift

```diff
@@ -1,6 +1,6 @@
 // This source file is part of the Swift.org open source project
 //
-// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
```

Comment:
Is this really necessary?

Reply:
Yes. Copyrights should be updated when files are updated.